extensions/2.0/Khronos/KHR_materials_sheen/README.md

```diff
@@ -0,0 +1,237 @@
+# KHR\_materials\_sheen
+
+## Contributors
+
+* Sebastien Vandenberghe, Microsoft, [@sebavan](https://github.com/sebavan)
+* Alexis Vaginay, Microsoft, [@Popov72](https://github.com/Popov72)
+* Tobias Häußler, Dassault System, [@proog128](https://github.com/proog128)
+* Ed Mackey, Analytical Graphics, Inc.
+* Romain Guy, Google, [@romainguy](https://github.com/romainguy)
+* Mike Bond, Adobe, [@MiiBond](https://github.com/MiiBond)
+* Don McCurdy, Google, [@donmccurdy](https://twitter.com/donrmccurdy)
+* Gary Hsu, Microsoft, [@bghgary](https://twitter.com/bghgary)
+* Eliphas NUIT, Dassault System, [@EliphasNUIT](https://github.com/EliphasNUIT)
```

Comment:
I would prefer if my real name, Jekfer Bichon, were used here instead of my alias.

Reply:
All good and changed, sorry about that. I did not find it when filling the section :-) Thanks for checking.